bin/run_ci.mjs

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env node
```

Comment:
This repository's ESLint configuration is used both for our other repositories and for this repository itself.

We cannot change the ESLint configuration to make it assume that `*.js` are pure ES modules yet until all our repositories have migrated.

Because of this, we need to use `*.mjs` files in this repository for the moment. Therefore, this PR renames several `*.js` to `*.mjs`.